app/javascript/packs/formatted-fields.js

```diff
@@ -0,0 +1,23 @@
+import Cleave from 'cleave.js';
+
+const SELECTOR_CONFIGS = {
```

Comment:
Note this excludes `.dob` from `form-field-format.js`, which no longer exists as of #4852.

Reply:
In 540671991, I also removed this from `form-validation.js`, along with `state_id_number` which was similarly removed as of #3682.